eth/main.cpp

```diff
@@ -1062,7 +1062,16 @@ int main(int argc, char** argv)
 
     web3.setIdealPeerCount(peers);
     web3.setPeerStretch(peerStretch);
-//  std::shared_ptr<eth::BasicGasPricer> gasPricer = make_shared<eth::BasicGasPricer>(u256(double(ether / 1000) / etherPrice), u256(blockFees * 1000));
+
+    // allow any gasPrice in testing mode
+    if (testingMode)
+    {
+        askPrice = 0;
+        bidPrice = 0;
+    }
+    //  std::shared_ptr<eth::BasicGasPricer> gasPricer =
+    //  make_shared<eth::BasicGasPricer>(u256(double(ether / 1000) / etherPrice), u256(blockFees *
+    //  1000));
```

Comment:
Why the commented out code ? It may be better to remove it, if it's no longer useful.

Reply:
this commented line was there for ages. @pirapira @chfast  do you approve removal?

Reply:
Sure.

Reply:
Yes, remove.